test/detect/typescript/default.txt

```diff
@@ -2,6 +2,7 @@ class MyClass {
   public static myValue: string;
   constructor(init: string) {
     this.myValue = init;
+    return <div className="test">jsx syntax</div>;
```

Comment:
Please also add a test with a self-closing tag as that seems to be a big issue with our React/JSX matching feature for this...